libraries/botbuilder-dialogs/tests/componentDialog.test.js

```diff
@@ -3,21 +3,21 @@ const { Dialog, DialogReason, DialogSet, DialogTurnStatus, ComponentDialog, Wate
 const assert = require('assert');
 
 function simpleStepContextCheck(step) {
-    assert(step, `step not found.`);
-    assert(typeof step === 'object', `step is not a WaterfallStepContext.`);
+    assert(step, 'step not found.');
+    assert(typeof step === 'object', 'step is not a WaterfallStepContext.');
 }
 
 describe('ComponentDialog', function () {
     this.timeout(5000);
 
     it('should set initial dialog to be first dialog added via addDialog()', function () {
-        const simpleWaterfall = new WaterfallDialog('simpleWaterfall', [async (step) => {}]);
+        const simpleWaterfall = new WaterfallDialog('simpleWaterfall');
```

Comment:
This look like a change in the code, and despite the code seems to do nothing I just want to double check this one. Are all tests passing?